DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: azuremlsdk
 Type: Package
 Title: Interface to the 'Azure Machine Learning' 'SDK'
-Version: 0.6.85
+Version: 0.8.0
```

Comment:
why did we jump from 0.6.85 to 0.8.0 rather than 0.7.0? I may have missed a conversation about this.

Reply:
we are using python sdk==1.8.0 as default version in install_azureml(), so, I made it 0.8.0 to be consistent with it.